gsoc/suborg_guidelines.md

```diff
@@ -13,20 +13,22 @@ slots we will get till the student selection process ends. However,
 it's our intention to keep the process fair for the suborganisations
 and students.
 
-## I want to have to become a sub-org admin, what do I do?
+## I want to become a sub-org admin, what do I do?
 
-First you need to be a member of the OpenAstronomy team. That process
+First, we recommend you to carefully read [Google's notes for first year organizations][Google-notes],
```

Comment:
point to the psf guidelines here, too. https://python-gsoc.org/mentors.html#sub-orgs, or in fact the whole mentors page is a good resource

Reply:
awesome!